### DISCUSSION TITLE
Using ImagePrompt, PyraCanny or CPDS makes fooocus crash

### DISCUSSION THREAD

**Discussion by FREAK-ish** [lllyasviel/Fooocus#2534] [Q&A] [answered]
So since the last update, every time I use the ImagePrompt, PyraCanny, or CPDS and press generate image, fooocus, it goes wild, gives an error message, and can't function with it any longer. Only after restarting, fooocus will work again but if I try the ImagePrompt, PyraCanny, or CPDS, the same thing will happen. Does anyone have this problem?

**Comment by mashb1t** [comment]
Please check your resources in task manager or similar for resource exhaustion and monitor if there are terminal log errors.

**Reply by FREAK-ish** [reply]
in the browser, after I click generate the message says, - Error, connection timed out...

And in Stability Matrix the part of the error is:

Requested to load SDXLClipModel

Requested to load GPT2LMHeadModel

Loading 2 new models

Total time: 404.88 seconds

[Fooocus Model Management] Moving model(s) has taken 1.50 seconds

1006

Exception in thread Thread-2 (worker):

Traceback (most recent call last):

File "threading.py", line 1016, in _bootstrap_inner

File "threading.py", line 953, in run

File "C:\Users\User\AppData\Roaming\StabilityMatrix\Packages\Fooocus\modules\async_worker.py", line 897, in worker

generate_image_grid = task.args.pop(0)

IndexError: pop from empty list

----------------------------------------------------------------------------------------------------------------------

in task manager appears this...
![Sem Título-1](https://github.com/lllyasviel/Fooocus/assets/148283905/3e45e98a-6a8b-4c54-af5e-0311d97404bb)

**Reply by mashb1t** [reply, marked as answer]
ah ok, this has already been reported here: https://github.com/lllyasviel/Fooocus/issues/2527

**Reply by FREAK-ish** [reply]
Can I get the file from somewhere? cause I don`t understand nothing about coding.

**Reply by mashb1t** [reply]
Here https://github.com/lllyasviel/Fooocus/pull/2537/files or in the develop branch.

**Reply by nadora35** [reply]
i have same issue . download tow file and same issue 
![Untitled](https://github.com/lllyasviel/Fooocus/assets/57017061/8bbd76ff-5b46-4f69-ad8d-52508cecf84e)
